admin/js/posts_report.js

```diff
@@ -1,13 +1,13 @@
 /* global jQuery, wp */
 
-var postCollection,
-  pageCollection,
-  postsView,
-  pagesView;
+let postCollection;
+let pageCollection;
+let postsView;
+let pagesView;
 
-var p4 = p4 || {};
-var p4_data = p4_data || {};
-var wp = window.wp || {};
+const p4 = p4 || {};
+const p4_data = p4_data || {};
```

Comment:
I think we define this `p4_data` in global scope in another case.
I get this error locally

```
Uncaught SyntaxError: Identifier 'p4_data' has already been declared
```

Reply:
hmm or maybe not in master-theme, but in another plugin

Reply:
@kirdia @comzeradd sorry for the delay, I added a commit to move the declarations into the closure and use "window." to reference the globals, since JS was complaining about these `const p4 = p4 || {};` statements, which seemed to trigger a "Cannot access 'p4' before initialization" exception. Apparently, you can't use the null coalesce operator with `let` and `const`, didn't know that, lesson learned. 
The posts report looks fine with this change.